presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -4962,6 +4962,74 @@ public void testShowColumnMetadata()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testRoleGrants()
+    {
+        assertUpdate("CREATE ROLE test_role_grant1");
```

Comment:
Can you test different users here? Like admin who sees everything, user that sees only his role grants and user that does not see anything because has no role grants?